build.gradle

```diff
@@ -58,7 +58,7 @@ subprojects {
     version = scmVersion.version
 
     dependencies {
-        api group: 'org.springframework', name: 'spring-web', version: '5.3.6'
+        api group: 'org.springframework', name: 'spring-web', version: '6.0.2'
```

Comment:
Might want to consider using

io.micrometer v1.10.3 to be in-line with Spring Boot Dependency versions:
https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/3.0.2/spring-boot-dependencies-3.0.2.pom

Also, for awareness, okhttp version is now on v4.10.0 as well.